R/slide.R

```diff
@@ -6,7 +6,8 @@
 #'
 #' @param x The `epi_df` object under consideration.
 #' @param f Function or formula to slide over variables in `x`. To "slide" means
-#'   to apply a function or formula over a running window of `n` time steps
+#'   to apply a function or formula over a running window of `before`
```

Comment:
I'd rewrite this as:
> ...over a running window of time steps. The window is determined by the `before` and `after` parameters described below. One time step is typically one day or one week; see details for more explanation. ...

Reply:
+1, but suggest "running" -> "rolling".  (I think "running" would refer to an expanding window with the left side anchored at the beginning of the time series.)

Reply:
Fixed.